pkg/kubectl/cmd/apply/apply.go

```diff
@@ -194,14 +195,15 @@ func NewCmdApply(baseName string, f cmdutil.Factory, ioStreams genericclioptions
 func (o *ApplyOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
 	o.ServerSideApply = cmdutil.GetServerSideApplyFlag(cmd)
 	o.ForceConflicts = cmdutil.GetForceConflictsFlag(cmd)
+	o.FieldManager = cmdutil.GetFieldManagerFlag(cmd)
 	o.DryRun = cmdutil.GetDryRunFlag(cmd)
 
 	if o.ForceConflicts && !o.ServerSideApply {
-		return fmt.Errorf("--force-conflicts only works with --server-side")
+		return fmt.Errorf("--experimental-force-conflicts only works with --experimental-server-side")
```

Comment:
I'm not quite sure why you combined the rename with the other part of this PR :)

Reply:
Clayton wanted me to rename the `--field-manager` to `--experimental-field-manager`, and I couldn't really sleep with the inconsistency so ... :-)

Reply:
SGTM